src/client/components/App/index.css

```diff
@@ -0,0 +1,4 @@
+Iframe {
+    width: 800px;

```

Comment:
@kilometerskang Having fixed px ratios is not great for mobile viewing. Can you try to make the iframe responsive? 

See https://www.w3docs.com/snippets/css/how-to-create-a-responsive-iframe-with-css.html